modules/managed-cluster-notification-troubleshooting.adoc

```diff
@@ -0,0 +1,20 @@
+[id="troubleshoot_{context}"]
+= Troubleshooting
+
+.If you are not receiving cluster notification emails
+* Ensure that email from sent from `@redhat.com` addresses is not filtered out of your email inbox.
+* Ensure that your correct email address is listed as a notification contact for the cluster.
+* Ask the cluster owner or administrator to add you as a notification contact:
+ifdef::openshift-rosa[]
+xref:../rosa-cluster-notifications.adoc#cluster-notification-emails_rosa-cluster-notifications[Cluster notification emails].
+endif::openshift-rosa[]
+ifdef::openshift-dedicated[]
+xref:../osd-cluster-notifications.adoc#cluster-notification-emails_osd-cluster-notifications[Cluster notification emails].
+endif::openshift-dedicated[]
+
+.If your cluster does not receive notifications
+* Ensure that your cluster is registered to the Hybrid Cloud Console: link:https://console.redhat.com/openshift/register[Register a cluster]
+* Ensure that you are able to access `api.openshift.com`.
```

Comment:
I think the intent of this is to check connectivity from the cluster to api.openshift.com, needs reworded?

Reply:
Reworded to "Ensure that your cluster can access resources at `api.openshift.com`." Let me know if you'd suggest something else!

Reply:
Full list may be useful to link? 

https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_prerequisites

https://docs.openshift.com/dedicated/osd_planning/aws-ccs.html#osd-aws-privatelink-firewall-prerequisites_aws-ccs